src/Features/Core/Portable/PullMemberUp/MemberAndDestinationValidator.cs

```diff
@@ -29,15 +29,19 @@ public static bool IsDestinationValid(Solution solution, INamedTypeSymbol destin
 
         public static bool IsMemberValid(ISymbol member)
         {
+            if (member.IsImplicitlyDeclared)
+            {
+                return false;
+            }
+
             // Static, abstract and accessiblity are not checked here but in PullMembersUpOptionsBuilder.cs since there are
             // two refactoring options provided for pull members up,
             // 1. Quick Action (Only allow members that don't cause error)
             // 2. Dialog box (Allow modifers may cause error and will provide fixing)
             return member switch
             {
                 IMethodSymbol methodSymbol => methodSymbol.MethodKind == MethodKind.Ordinary,
-                IFieldSymbol fieldSymbol => !fieldSymbol.IsImplicitlyDeclared,
-                _ => member.IsKind(SymbolKind.Property) || member.IsKind(SymbolKind.Event),
```

Comment:
Are there tests for implicitly declared fields? If not, I think it might be best to revert this change, as you'll have to remove the check on line 32 in future, to do record parameters, and it would be easy to forget to put this back.